src/Symfony/Component/Process/Pipes/WindowsPipes.php

```diff
@@ -52,8 +52,7 @@ public function __construct($disableOutput, $input)
                 Process::STDERR => tempnam(sys_get_temp_dir(), 'sf_proc_stderr'),
```

Comment:
> prefix
The prefix of the generated temporary filename.
Note: Windows uses only the first three characters of prefix.

So the prefix would be the same for stdout and stderr on windows, which defeats the purpose. I propose to change the prefix to be different, e.g. `out` and `err`